### PR TITLE
Fix for PowerShell

### PR DESCRIPTION
**Pull request by jakub-suliga**
Fixed Problem with the trialing whitespaces and added requirements in the Readme

### CODE REVIEW

File: lib.ps1 [hunk 1]
```diff
@@ -1,7 +1,7 @@
-$GIT_FOLDER = "platform-contributor-all\git"
+$GIT_FOLDER = "D:\Dev\Eclipse\eclipse-contrib\git"
```
Comment (fedejeanne) [inline lib.ps1]: Please don't put your own values here, they should be kept local (for your own usage only)

File: lib.ps1 [hunk 2]
```diff
@@ -1,7 +1,7 @@
-$GIT_FOLDER = "platform-contributor-all\git"
+$GIT_FOLDER = "D:\Dev\Eclipse\eclipse-contrib\git"
 
 # GitHub username
-$GH_USERNAME = "fedejeanne"
+$GH_USERNAME = "Kuba3105"
```
Comment (fedejeanne) [inline lib.ps1]: Dito: must remain local

File: README.md [hunk 3]
```diff
@@ -1,2 +1,8 @@
 # eclipse-scripts
 Useful scripts to work with Eclipse
+
+#requirements
+1. Download https://cli-github-com.translate.goog/?_x_tr_sl=en&_x_tr_tl=de&_x_tr_hl=de&_x_tr_pto=sc
```
Comment (fedejeanne) [inline README.md]: Thanks! I totally forgot this one.

**Review by fedejeanne — CHANGES_REQUESTED** [review]
Some values must remain in your local copy, don't push them to this repo please.

File: lib.ps1 (hunk 1 above)
Comment (jakub-suliga) [reply lib.ps1]: Solved in 46dcd3b37f4b18beefd6db44ad98e97e68501248

File: lib.ps1 (hunk 2 above)
Comment (jakub-suliga) [reply lib.ps1]: Solved in 46dcd3b37f4b18beefd6db44ad98e97e68501248

**Comment by fedejeanne** [comment]
@Kuba3105 you may close this PR since you created the new one: #2

**Pull request closed without merging**